libc/cmake/caches/baremetal_common.cmake

```diff
@@ -0,0 +1,17 @@
+# Expects target triple to be passed as `ARCH_TRIPLE`
```

Comment:
I'd call the variable `RUNTIMES_TARGET_TRIPLE` since we want to use the `RUNTIMES_` prefix for all runtimes-build related variables.